roles/inject_openshift_kube_objects/tasks/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: "Create the kube_objects directory"
+  shell: "mkdir -p {{ kube_objects_dir }}"
+
+- name: "Get the CVP private key"
+  copy:
+    src: "{{ ssh_private_key }}"
+    dest: "{{ kube_objects_dir }}/private_key"
+    mode: 0600
+  no_log: true  
+
+- name: "Decode the symmetric key received thru UMB message"
+  shell: "echo \"{{ isv_symm_key }}\" | openssl enc -d -base64 > {{ kube_objects_dir }}/dec_sym_key"
```

Comment:
`dec_sym_key` might be a bit confusing. Sure it i decoded but just *base64 decoded*
But it is not decrypted symetric key